src/LedgerLivePlatformSDK/index.ts

```diff
@@ -222,6 +222,17 @@ export default class LedgerLivePlatformSDK {
     });
   }
 
+  /**
+   * Let the user sign the provided message through Ledger Live
+   * @param accountId - Ledger Live id of the account (Ethereum only)
+   * @param message - Message the user should sign
+   *
+   * @returns Message signed
+   */
+  async signMessage(accountId: string, message: string): Promise<string> {
+    return this._request("message.sign", { accountId, message });
```

Comment:
We might actually want to hash the `message` before sending it over, and convert it back to clear text on LL, to reduce payload size

Reply:
In the end it's an IPC communication. Don't you think it is a premature optimization?
I wonder what can be the size of a message in this method versus the size of a transaction payload in the signTransaction call 🤔

Reply:
A message can be a string of an arbitrary length, up to `2^53 - 1` character (cf. [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/length#:~:text=ECMAScript%202016%20(ed.%207)%20established%20a%20maximum%20length%20of%202%5E53%20%2D%201%20elements)), each character being between 2 to 5 bytes or more (cf. [this](https://stackoverflow.com/questions/2219526/how-many-bytes-in-a-javascript-string#:~:text=ES6%20and%20later%3A%202%20bytes%20per%20character%2C%20or%205%20or%20more%20bytes%20per%20character)), so the message can be at most `(2^53 - 1) * 40` bits, whereas popular hash functions generate outputs between 160 and 512 bits (cf. [this](https://www.tutorialspoint.com/cryptography/cryptography_hash_functions.htm#:~:text=Hash%20function%20with%20n%20bit%20output%20is%20referred%20to%20as%20an%20n%2Dbit%20hash%20function.%20Popular%20hash%20functions%20generate%20values%20between%20160%20and%20512%20bits.))

Reply:
I'm not sure I understand here, hashing is not to be mixed with compression, there are no way to "convert it back to clear text" 🤔